invokeai/frontend/web/src/services/api/types.d.ts

```diff
@@ -58,14 +59,17 @@ export type DiffusersModelConfig =
 export type CheckpointModelConfig =
   | components['schemas']['StableDiffusion1ModelCheckpointConfig']
   | components['schemas']['StableDiffusion2ModelCheckpointConfig']
+  | components['schemas']['StableDiffusion2ModelDiffusersConfig'];
+export type OnnxModelConfig = components['schemas']['ONNXStableDiffusion1ModelConfig']
```

Comment:
This doesn't look correct, it's added a diffusers type to the checkpoint union and sdxl to the onnx union